src/ClientHandle.cpp

```diff
@@ -2907,9 +2904,9 @@ void cClientHandle::AddWantedChunk(int a_ChunkX, int a_ChunkZ)
 	
 	LOGD("Adding chunk [%d, %d] to wanted chunks for client %p", a_ChunkX, a_ChunkZ, this);
 	cCSLock Lock(m_CSChunkLists);
-	if (std::find(m_ChunksToSend.begin(), m_ChunksToSend.end(), cChunkCoords(a_ChunkX, a_ChunkZ)) == m_ChunksToSend.end())
+	if (m_ChunksToSend.find(cChunkCoords(a_ChunkX, a_ChunkZ)) == m_ChunksToSend.end())
```

Comment:
I understand that since normal maps have unique keys, count returns 0 or 1, which may look better in comparison to comparing to containername.end().

However, some advise that find can be faster, even when count is aliased to find in sets and maps, because of less function calls.